neqo-http3/src/connection.rs

```diff
@@ -476,6 +471,53 @@ impl Http3Connection {
         self.recv_streams.clear();
     }
 
+    fn handle_stream_manipulation_output<U>(
```

Comment:
Sorry, I found this function is not easy to understand for me. Could you add some comments?

Reply:
I have added some explanations.